src/value.cpp

```diff
@@ -390,7 +391,7 @@ std::string CommentValue::comment(const char* encoding) const {
   }
   bool bAscii = charsetId() == undefined || charsetId() == ascii;
   // # 1266 Remove trailing nulls
-  if (bAscii && c.find('\0') != std::string::npos) {
+  if (bAscii && Internal::contains(c, '\0')) {
     c.resize(c.find('\0'));
```

Comment:
Perhaps refactor this one differently so find() is potentially not evaluated twice?

Reply:
Done